docs/jwst/associations/generator.rst

```diff
@@ -52,20 +50,18 @@ have. For example, if a JWST coronagraphic observation was performed, but the
 related PSF observation failed, the coronagraphic association would be marked invalid.
 
 Once validation is complete, :py:func:`~jwst.associations.generate` returns a
-2-tuple. The first item is a list of the associations created. The second item
```

Comment:
I looked at the code and I don't see the second item, so I removed it from the doc. Did I miss something here?

https://github.com/spacetelescope/jwst/blob/6cb593526baa0f83b6ec62fcba1321b00b6cd5b9/jwst/associations/generator/generate.py#L114